### PR TITLE
dbus-services: adjust bluez whitelisting to /usr (bsc#1199207)

### PR DESCRIPTION
**Pull request by mgerstner**
On this occasion also introduce digest based whitelisting entries.

### CODE REVIEW

File: configs/openSUSE/dbus-services.toml [hunk 1]
```diff
@@ -285,11 +285,21 @@ nodigests = [
...
-    "/etc/dbus-1/system.d/bluetooth.conf",
-    "/usr/share/dbus-1/system-services/org.bluez.service",
+note = "Provides bluetooth stack access via D-Bus"
+bugs = ["bsc#768062", "bsc#1199207"]
+digests = [
+    {
+        path = "/usr/share/dbus-1/system.d/bluetooth.conf",
+        algorithm = "sha256",
+        digester = "xml",
+        hash = "529151093a482e8223738f86bec3dac4c23733c38fba1f318cc6130b5acaebfe"
```
Comment (wfrisch) [inline configs/openSUSE/dbus-services.toml:295]: The current hash in openSUSE:Factory/bluez and also Base:System/bluez is `c211c4412f39dec5dd93c3da9a8b4bbeec37825fdb74d5fd8efd23573f11f1ed`

File: configs/openSUSE/dbus-services.toml [hunk 2]
```diff
@@ -285,11 +285,21 @@ nodigests = [
...
+    },
+    {
+        path = "/usr/share/dbus-1/system-services/org.bluez.service",
+        algorithm = "sha256",
+        digester = "shell",
+        hash = "8004c707a071bab12c20f03d0f63d3edea59bbc4d1697ebec44b12fa58f81fea"
```
Comment (wfrisch) [inline configs/openSUSE/dbus-services.toml:301]: OK

**Review by wfrisch — CHANGES_REQUESTED** [review]
The sha256 hash of `bluetooth.conf` doesn't match Base:System/bluez.
It should be, IMHO:
`c211c4412f39dec5dd93c3da9a8b4bbeec37825fdb74d5fd8efd23573f11f1ed`

File: configs/openSUSE/dbus-services.toml (hunk 1 above)
Comment (mgerstner) [reply configs/openSUSE/dbus-services.toml:295]: It looks like you digested the plain file, this digest is for the XML filtered file, please use `tools/get_whitelisting_digest.py` from the repository to compare digests.

Comment (wfrisch) [reply configs/openSUSE/dbus-services.toml:295]: Yes, I digested the plain file. Sorry for the noise.
`529151093a482e8223738f86bec3dac4c23733c38fba1f318cc6130b5acaebfe` is correct.

**Comment by wfrisch** [comment]
Nevermind. The hashes are OK.

**Review by wfrisch — APPROVED** [review]

**Pull request merged**